rct229/rules/section21/section21rule3.py

```diff
@@ -0,0 +1,54 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.get_baseline_system_types import get_baseline_system_types
+from rct229.utils.assertions import getattr_
+
+APPLICABLE_SYS_TYPES = ["SYS-1", "SYS-5", "SYS-7", "SYS-11.2", "SYS-12", "SYS-1A", "SYS-7A", "SYS-11.2A", "SYS-12A"]
+FLUID_LOOP = schema_enums["FluidLoopOptions"]
+
+
+class Section21Rule3(RuleDefinitionListIndexedBase):
+    """Rule 3 of ASHRAE 90.1-2019 Appendix G Section 23 (Hot water loop)"""
+    def __init__(self):
+        super(Section21Rule3, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section21Rule3.HeatingFluidLoopRule(),
+            index_rmr="baseline",
+            id="21-3",
+            description="Heating hot water plant capacity shall be based on coincident loads.",
+            list_path="ruleset_model_instances[0].fluid_loops[*]",
```

Comment:
I think you need
```
rmr_context="ruleset_model_instances/0"
list_path="fluid_loops[*]"
```
_rmr_context_ provides the correct context for _is_applicable_  while _list_path_ provides the correct context for _listfilter_ and HeatingFludiLoopRule.